src/improvements/tasks/eth/013-gas-params-op/README.md

```diff
@@ -0,0 +1,44 @@
+# 013-gas-params-op: (Ethereum) Increase gas target : OP Mainnet
+
+Status: [READY TO SIGN]
+
+## Objective
+
+Sets the following gas params in the SystemConfig contract for OP Mainnet:
+* gasLimit: 40_000_000
+* eip1559_elasticity: 2
+
+This has the effect of setting the gasTarget to 20Mgas/block.
+
+1. OP Mainnet
+
+### Timing
```

Comment:
Some of these things have been removed from the latest `just new task` command. And some things have been added (like the normalization hash section in the VALIDATION.md file - I don't think you need this for your task though).

Reply:
Ah I see. I ended up copying the README.md and VALIDATION.md from my previous 008-gas-params-op task since this one is nearly identical. Let me know if I need to adapt this to a new format though. Sounds like maybe not necessary?